lib/validation/validation.go

```diff
@@ -0,0 +1,30 @@
+// Package validation provides miscellaneous standalone validation utilities
+// for validating inputs. For the most part, these utilities lived somewhere
+// else in the codebase before they were moved here when a more common location
+// was warranted.
```

Comment:
I think we can keep the comment terse here
```suggestion
// Package validation provides standalone utility functions 
// for validating inputs. 
```